Uyanda.Coffee.Application/Features/BeverageManagement/Services/IBeverageManagementService.cs

```diff
@@ -14,5 +14,9 @@ public interface IBeverageManagementService
         Task<PurchaseResult> PurchaseAsync(PurchaseCommand purchase);
```

Comment:
Add a supporing method, say _GetCustomerPoints_ who's query parameter has the customer's phone numbers. And result class has both the number of points the customer has and the number of coffee cups the customer can settle with the points.